mithril-stm/src/protocol/mod.rs

```diff
@@ -0,0 +1,62 @@
+mod aggregate_signature;
+mod eligibility_check;
+mod error;
+mod key_registration;
+mod parameters;
+mod participant;
+mod single_signature;
+
+pub use aggregate_signature::{
+    AggregateSignature, AggregateSignatureType, AggregateVerificationKey, BasicVerifier, Clerk,
+};
+pub use error::*;
+pub use key_registration::*;
+pub use parameters::*;
+pub use participant::{Initializer, Signer, VerificationKey, VerificationKeyProofOfPossession};
+pub use single_signature::{SingleSignature, SingleSignatureWithRegisteredParty};
```

Comment:
It's probably better to control exactly what we make public:
```suggestion
pub use aggregate_signature::{
    AggregateSignature, AggregateSignatureType, AggregateVerificationKey, BasicVerifier, Clerk,
};
pub use error::{
    AggregateSignatureError, AggregationError, MultiSignatureError, RegisterError, SignatureError,
};
pub use key_registration::{ClosedKeyRegistration, KeyRegistration};
pub use parameters::Parameters;
pub use participant::{Initializer, Signer, VerificationKey, VerificationKeyProofOfPossession};
pub use single_signature::{SingleSignature, SingleSignatureWithRegisteredParty};
```